03-when-vector-data-dont-line-up-CRS.Rmd

```diff
@@ -342,22 +339,22 @@ into the `R` console.
 
 ## Reproject Vector Data
 
-Now we know our data are in different `CRS`. To address this, we have to modify
-or `reproject` the data so they are all in the **same** `CRS`. We can use
-`spTransform` function to reproject our data. When we reproject the data, we
-specify the `CRS` that we wish to transform our data to. This `CRS` contains
-the `datum`, units and other information that `R` needs to `reproject` our data.
+Now we know our data are in different CRS. To address this, we have to modify
+or **reproject** the data so they are all in the **same** CRS. We can use
```

Comment:
hmmm - curious why you are using CRS without code brackets. We want to emphasize it. the function is lower case i think though. if we don't use code brackets it may be worth bolding. but i'm inclined to stick with code brackets.